### PR TITLE
Incoming ert command inspection

### PR DESCRIPTION
**Pull request by chienwei-lan**

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c [hunk 1]
```diff
@@ -976,18 +1003,14 @@ static inline int process_ert_rq(struct xocl_ert_user *ert_user)
...
-				ecmd->status = KDS_ABORT;
-				list_move_tail(&ecmd->list, &ert_user->cq);
-				--ert_user->num_rq;
-				++ert_user->num_cq;
-				continue;
-			}
-		} else if (cmd_opcode(ecmd) == OP_START)
-			BUG_ON(ert_user->ctrl_busy || !ert_user->config);
+		if (ert_cmd_inspection(ert_user, ecmd)) {
+			ERTUSER_ERR(ert_user, "%s bad cmd, opcode: %d\n", __func__, cmd_opcode(ecmd));
```
Comment (mamin506) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c:1007]: Not sure if this should be a debug message.

**Review by mamin506 — APPROVED** [review]
Overall, this looks good to me.

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c (hunk 1 above)
Comment (chienwei-lan) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c:1007]: once there is any cmd its opcode we can't recognize, we should have some clues in dmesg

or does kds already report something?

Comment (mamin506) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c:1007]: KDS doesn't report something. I think we can print error here. This is okay.

**Pull request merged**